stylesheets/style.css

```diff
@@ -301,7 +296,64 @@ section {
     width: 100%;
     text-align: center; }
     footer:last-child {
-      margin-right: 0; } }
+      margin-right: 0; }
+
+  .key-dates {
+    position: relative; }
+
+  .key-date {
+    display: block; }
+    .key-date h4 {
```

Comment:
Because you're nesting, you don't need to include ```.key-date``` here. Let's remove these additional styles.